### PR TITLE
add BOM support for csv file

### PR DESCRIPTION
**Pull request by fanjinfei**

### CODE REVIEW

**Comment by fanjinfei** [comment]
ready for review

**Comment by fanjinfei** [comment]
@wardi You already have PR for csv BOM format. Should I close this?

**Comment by thriuin** [comment]
@wardi Can you review in the next day or two - I would wait until Monday to deploy. Is there a staging area where the whole process can be tested?

File: ckanext/recombinant/read_csv.py [hunk 1]
```diff
@@ -17,6 +18,12 @@ def csv_data_batch(csv_path, chromo):
     current_owner_org = None
 
     with open(csv_path, 'rb') as f:
+        first3bytes = f.read(3)
+        if first3bytes == codecs.BOM_UTF8:
+            f.seek(3)
```
Comment (wardi) [inline ckanext/recombinant/read_csv.py]: this is a NOP :-) you could save a couple lines by skipping it

**Comment by wardi** [comment]
@thriuin yes, we have staging reg+portal now but we still need to set up the scripts that push PD like we have on the prod reg+portal.

**Comment by thriuin** [comment]
@wardi Thanks!

**Pull request merged**